R/scripts/create_filter.R

```diff
@@ -20,13 +21,12 @@ incid_data_list <- covidImportation::get_incidence_data(
   first_date = ISOdate(2019,12,1),
   last_date = as.POSIXct(lubridate::ymd(config$end_date)),
   update_case_data = TRUE,
-  case_data_dir = file.path('importation',config$spatial_setup$setup_name,"case_data"),
+  case_data_dir = file.path("data", config$spatial_setup$setup_name,"case_data"),
```

Comment:
This isn't right.  Not sure what you're going for here though.  Do you maybe mean config$spatial_setup$base_path instead of "data"?

Reply:
okay - I wasn't sure what the correct file system structure was.

But also I'm not sure this script should be committed because it didn't work with the CA data... Is there a way to exclude this commit from the pull request & merge? It requires further testing and adjustment. I didn't realize that this commit was in here...

Reply:
Not in an obvious way.  I could revert the commit in question, but don't know what else was part of it.  Might be best to just fix the script?

Reply:
I'm not sure what is wrong with it.. unfortunately. None of the simulations passed the filter when I used this filter and I suspect is a geoid ordering issue again

Reply:
Maybe we should instead re-commit the old version of the file?

Reply:
i can take a look at the file, give me a moment.